src/bundleList.js

```diff
@@ -127,6 +131,13 @@ function initBundleBuckets(roles) {
 function sortBundleByBuckets(roles, bundle, bundleBuckets) {
   roles.forEach((role) => {
     if (bundle.indexOf('-' + role + '-') !== -1) {
+
+      // skip invalid bundle names
+      if( !validBundleRegex.test( bundle ) ){
+        console.error( 'skip bundle', bundle );
```

Comment:
The only issue with `info()` is that it logs to stderr instead of stdout.

I will create a separate ticket to migrate these scripts to using `pelias/logger`